x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/utils/deprecation.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Logger } from 'src/core/server';
+
+/**
+ * Logs usages of a deprecated endpoint
+ *
+ * @param logger System logger
+ * @param path Deprecated endpoint path
+ */
+export const logDeprecatedEndpoint = (logger: Logger, path: string) => {
+  logger.info(`${path} API is deprecated. Please refer to the documentation for alternatives.`);
```

Comment:
It would be great if it either was referring to a specific page in the docs, or saying right away what the user can do with this. All Kibana config deprecation messages suggest how the user can fix the config.

I'm thinking about something like:

`Deprecated endpoint: ${path} API is deprecated since v${version}. ${suggestion}`
`Deprecated endpoint: /api/detection_engine/rules/_bulk_delete API is deprecated since v8.2.0. Please use the /api/detection_engine/rules/_bulk_action API instead`

It would be great to pass the same message to the deprecation header.

Reply:
Oh sorry, one more thing: should be `logger.warn` I guess?

Reply:
Updated message and log level 👍